overlord/snapstate/backend_test.go

```diff
@@ -431,7 +431,10 @@ func (f *fakeStore) SnapAction(ctx context.Context, currentSnaps []*store.Curren
 
 		// refresh
 
-		cur := curByID[a.SnapID]
+		cur := curByInstanceName[a.InstanceName]
+		if cur == nil {
+			return nil, fmt.Errorf("internal error: no matching current snap for %q", a.InstanceName)
```

Comment:
This error reads a little bit weird, but I couldn't come up with a better idea (nor find similiar error case in the code, the closest I found was "internal error: current not found in sequence: %v %v" but it doesn't fit this context).